scaletest/templates/scaletest-runner/scripts/report.sh

```diff
@@ -27,7 +27,11 @@ server_version="$(jq -r '.version' <<<"${buildinfo}")"
 server_version_commit="$(jq -r '.external_url' <<<"${buildinfo}")"
 
 # Since `coder show` doesn't support JSON output, we list the workspaces instead.
-workspace_json="$(DRYRUN=0 coder list --all --output json | jq --arg workspace "${CODER_WORKSPACE}" --arg user "${CODER_USER}" 'map(select(.name == $workspace) | select(.owner_name == $user)) | .[0]')"
+# Use `command` here to bypass dryrun.
```

Comment:
nit: dry run?

Reply:
Are you suggesting typo correction or something else?

Reply:
yes, typo 👍 nitpick